spec/lib/alerts/dyk_nomination_monitor_spec.rb

```diff
@@ -57,6 +62,13 @@ def mock_mailer
       expect(Alert.last.email_sent_at).not_to be_nil
     end
 
+    it 'emails course creator' do
+      create(:courses_user, user_id: course_creator.id, course_id: course.id, role: 1)
+      allow_any_instance_of(AlertMailer).to receive(:alert).and_return(mock_mailer)
+      described_class.create_alerts_for_course_articles
+      expect(Alert.last.email_sent_at).not_to be_nil
```

Comment:
A better check here might be to make sure that the person to whom the email was sent matches up with the `course_creator`. Right now, it looks like you're just checking to make sure an email was sent. I imagine you could either take a look at other properties on the `Alert.last` object or expect `AlertMailer.alert` method to receive the `course_creator`.